screens/ImagesScreen/RenderImagePairs.jsx

```diff
@@ -17,17 +17,29 @@ const RenderImagePairs = changeProfileImage => {
   const {arrayImages} = useContext(GlobalDataContext);
   const maximum = 4;
   const minimum = 1;
+  const [minimumImageSize, setMinimumImageSize] = useState(1);
+  const [maximumImageSize, setMaximumImageSize] = useState(4);
```

Comment:
what are minimum and maximum above these states? if they are constant move out of this component, for naming use THIS_STYLE and also, you can use them in these states defaults